jib-core/src/main/java/com/google/cloud/tools/jib/image/json/ContainerConfigurationTemplate.java

```diff
@@ -54,6 +55,15 @@
 @JsonIgnoreProperties(ignoreUnknown = true)
 public class ContainerConfigurationTemplate implements JsonTemplate {
 
+  /**
+   * A combined date and time at which the image was created. Constant to maintain reproducibility
+   * and avoid Docker's weird "292 years old" bug.
+   *
+   * @see <a
+   *     href="https://github.com/GoogleContainerTools/jib/issues/341">https://github.com/GoogleContainerTools/jib/issues/341</a>
```

Comment:
I think it's worth exploring some of the options mentioned in the bazel build bugs also : https://groups.google.com/forum/#!topic/istio-dev/nuv6UaBFGDk (like allowing a user configurable time)? or an env variable. Perhaps file a followup bug for that.

Reply:
Filed #413